rz-sbc/git_patch.json

```diff
@@ -1,79 +1,88 @@
 {
 	"poky": {
 		"url": "https://git.yoctoproject.org/git/poky",
-		"branch": "",
-		"tag": "dunfell-23.0.26",
+		"branch": "styhead",
+		"tag": "",
 		"commit": "",
-		"patches": ["patches/poky/0001-meta-classes-esdk-explicitly-address-the-location-of.patch"],
-		"type": "git"
+		"patches": [],
+		"type": "git",
+		"enable": "true"
 	},
 	"meta-summit-radio": {
 		"url": "https://github.com/LairdCP/meta-summit-radio.git",
 		"branch": "lrd-11.39.0.x",
 		"tag": "",
 		"commit": "",
-		"patches": ["patches/meta-summit-radio/0001-rzsbc-summit-radio-pre-3.4-support-eSDK-build.patch",
-					"patches/meta-summit-radio/0002-rzsbc-summit-radio-pre-3.4-enable-usb-bt-support.patch"],
-		"type": "git"
+		"patches": ["patches/meta-summit-radio/0001-rz-sbc-Support-build-in-yocto-styhead-release.patch"],
+		"type": "git",
+		"enable": "true"
 	},
 	"meta-renesas": {
 		"url": "https://github.com/Renesas-SST/meta-renesas.git",
-		"branch": "dunfell/rz-sbc",
+		"branch": "styhead/rz-sbc-codec-porting",
```

Comment:
@tiennguyenuh , check this line. I think it should be "styhead/rz-sbc"

Reply:
I accidentally pushed the changes, updated in the commit: https://github.com/Renesas-SST/rz-build-scripts/commit/2d42b08bdfd6a1720e7e20976be316c94cfa4d74